go.mod

```diff
@@ -8,4 +8,5 @@ require (
 	github.com/sirupsen/logrus v1.2.0
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5 // indirect
+	golang.org/x/sys v0.0.0-20200909081042-eff7692f9009 // indirect
```

Comment:
Can you remove this dependency? You can run a `go mod tidy` and `go mod vendor` to clean it up.